commands/user.go

```diff
@@ -334,27 +334,24 @@ func userActivateCmdF(c client.Client, command *cobra.Command, args []string) er
 }
 
 func changeUsersActiveStatus(c client.Client, userArgs []string, active bool) {
-	users := getUsersFromUserArgs(c, userArgs)
-	for i, user := range users {
-		if user == nil {
-			printer.PrintError(fmt.Sprintf("can't find user '%v'", userArgs[i]))
-			continue
-		}
-
-		err := changeUserActiveStatus(c, user, userArgs[i], active)
-
+	users, err := getUsersFromArgs(c, userArgs)
+	if err != nil {
+		printer.PrintError(err.Error())
+	}
+	for _, user := range users {
+		err := changeUserActiveStatus(c, user, active)
```

Comment:
This can be compressed into:

```suggestion
		if err := changeUserActiveStatus(c, user, active); err != nil {
```